torch_topological/nn/multi_scale_kernel.py

```diff
@@ -0,0 +1,138 @@
+import torch
+
+from torch_topological.utils import wrap_if_not_iterable
+
+
+class MultiScaleKernel(torch.nn.Module):
+    # TODO: more detailed description
+    r"""Implement the multi-scale kernel between two persistence diagrams
+
+    This class implements the multi-scale kernel between two persistence
+    diagrams as defined by Reininghaus et al. as
+
+    .. math::
+        k_\sigma(F,G) = \frac{1}{8 \pi \sigma} \sum_{\substack{p \in F\\q \in G}} exp{-\frac{\|p-q\|^2}{8\sigma}} - exp{-\frac{\|p-\overline{q}\|^2}{8\sigma}}
+
+    where :math:`z=(z_1, z_2)` and :math:`\overline{z}=(z_2, z_1)`
+
+    Attributes
+    ----------
+    sigma : float
+        Scale parameter
+
+    Methods
```

Comment:
I'm a little bit hazy on REST conventions. Does this method have to be provided here? I was hoping that Sphinx would find the definitions automatically.

Reply:
As far as my experience with Sphinx goes, it is indeed smart enough to know without the methods being explicitly provided. The part is removed in the updated version.

Reply:
Very good, I appreciate it!